src/platform/wii/main.c

```diff
@@ -892,6 +899,20 @@ uint16_t _pollGameInput(struct mGUIRunner* runner) {
 			keys |= 1 << angles;
 		}
 	}
+	int a_l = PAD_TriggerL(0);
+	int a_r = PAD_TriggerR(0);
+	if (a_l > 0x50) {
+		keys |= 1 << GBA_KEY_L;
+	}
+	if (a_r > 0x50) {
+		keys |= 1 << GBA_KEY_R;
+	}
+	
+    if (m_reset)
+    {
+       runner->core->reset(runner->core);
```

Comment:
This is probably not the best place to do this. In fact, explicitly binding the reset button to game reset is a bit of a questionable choice, vs closing the game and going back to the menu, or any other of a handful of choices.